### ISSUE TITLE
Accept Excel files for metadata

### ISSUE THREAD

**Issue by emmahodcroft** [loculus-project/loculus#3334]
A _lot_ of people use Excel to handle and store metadata. Whether rightly or wrongly, it is what it is. We will lower the bar for upload significantly if we can support Excel metadata uploads. 

We would probably need to force people to set the dates into ISO still, unless Excel encodes whether dates are US format or not (which it well might) to avoid confusion of whether `04/07/2022` is 7 of April or 4 of July.

**Comment by anna-parker** [comment]
I fully agree, I think every beta tester we have talked to so far has raised this exact point, they use excel and would like more support for it

**Comment by fhennig** [comment]
Good point about the dates! I did the PoC now https://github.com/loculus-project/loculus/pull/3426 and looked into the docs of the library I used. It seems like excel internally stores date as an offset, the stuff seen in the UI is not actually stored in the file verbatim (source: [SheetJS docs](https://docs.sheetjs.com/docs/csf/features/dates/#how-sheetjs-handles-dates-and-times)). The library seems to come with some date parsing support but I'm not sure which information you need to give it (i.e. which columns are dates?) -- would need to look into it more.
